apps/src/studio/studio.js

```diff
@@ -1184,6 +1184,20 @@ function handleActorCollisionsWithCollidableList (
             callHandler('whenGetAllItems');
           }
 
+          if (list == Studio.items) {
+            var className = collidable.className;
+            var itemCount = 0;
+            for (var j = 0; j < list.length; j++) {
+              if (className == list[j].className) {
+                itemCount++;
+              }
+            }
+
+            if (itemCount == 1) {
```

Comment:
Use `===` - possible copy or combine comment abovethat states `// NOTE: if items are allowed to move outOfBounds()...`